applications/finetune-quora-embeddings/helpers/models.py

```diff
@@ -0,0 +1,76 @@
+from typing import Literal, List
+from tqdm import tqdm
+import os
+from cohere import Client
+from openai import AsyncOpenAI
+import asyncio
+from sentence_transformers import SentenceTransformer
+
+model_types = Literal["OpenAI", "Cohere", "HuggingFace"]
+MODEL_TYPES_VALUES = {"HuggingFace", "OpenAI", "Cohere"}
+
+
+class EmbeddingModel:
```

Comment:
Model.from_cohere() where from_cohere is a classemethods
then all model has is a `embed` method

Reply:
class Factory():
     
     @classmethod
     def from_cohere():
          return Embeeding(cohere...)
     
     @classmethod
     def from_openai():
          return Embeeding(openai...)
     
     
  client = [
       Factory.from_cohere(), Factory.from_openai()
  ]
  
  for client in clients:
          embedding = client.embed()